### PR TITLE
add section about external SOLR install #17

### PR DESCRIPTION
**Pull request by mradamcox**
Adds a section to address #17.

### CODE REVIEW

**Comment by mradamcox** [comment]
@dl-maura Check out these changes and we can discuss here if there's more you would like to add/alter.

File: docs/installation/getting_started_developers.md [hunk 1]
```diff
@@ -55,6 +55,38 @@ $ rake engine_cart:server
...
+
+#### Configure the SOLR Core
+
+Once you have SOLR installed, you will need to create a new core and configure it for GeoBlacklight. How you create the core may depend on your installation method, but will likely be something like
+
+```bash
+$ bin/solr -c blacklight-core
+```
+
+Now rename/remove the core's `conf` directory and replace it with the `solr/conf` directory from GeoBlacklight: [github.com/geoblacklight/geoblacklight/tree/v4.0.0/solr/conf](https://github.com/geoblacklight/geoblacklight/tree/v4.0.0/solr/conf).
```
Comment (ewlarson) [inline docs/installation/getting_started_developers.md:70]: Let's change this URL from `tree/v4.0.0` to just `main`:
https://github.com/geoblacklight/geoblacklight/tree/main/solr/conf

That will help keep these instructions current regardless of latest release number.

**Review by ewlarson — CHANGES_REQUESTED** [review]
Just one little change request and I'll 🚢  it.

File: docs/installation/getting_started_developers.md (hunk 1 above)
Comment (mradamcox) [reply docs/installation/getting_started_developers.md]: No problem, change made.

**Comment by ewlarson** [comment]
Thanks @mradamcox. Merged!

**Pull request merged**